### DISCUSSION TITLE
How to disable ripple effect?

### DISCUSSION THREAD

**Discussion by chentao169** [quasarframework/quasar#13884] [General - Components / Directives / etc] [answered]
I wanna disable ripple effect when clicking the button or tab. I already tried to set `ripple="false"` in q-btn and q-tab. But it doesn't work

**Comment by pratik227** [comment, marked as answer]
https://codepen.io/Pratik__007/pen/vYRLOoW

You need to use `:ripple="false"`

**Reply by chentao169** [reply]
thank you!
